### DISCUSSION TITLE
Add new method startSurvey

### DISCUSSION THREAD

**Discussion by neburas** [devrnt/react-use-intercom#594] [Ideas]
Hi!

Are you planning to add the startSurvey method? I think it's similar to startTour

See https://developers.intercom.com/installing-intercom/docs/intercom-javascript#intercomstartsurvey-surveyid

Thanks!

**Comment by devrnt** [comment]
That's already available, see [v1.4.0](https://github.com/devrnt/react-use-intercom/releases/tag/react-use-intercom%404.1.0)
